src/component/container/LayerTreeClassic/LayerTreeClassic.tsx

```diff
@@ -31,14 +35,17 @@ export class LayerTreeClassic extends React.Component<LayerTreeClassicProps> {
   public static defaultProps: DefaultLayerTreeClassicProps = {
     extraLegendParams: {
       'LEGEND_OPTIONS': 'fontAntiAliasing:true;forceLabels:on;fontName:DejaVu Sans Condensed'
-    }
+    },
+    dispatch: undefined
```

Comment:
i think this is not a good default, as dispatch is usually a function. Maybe use `() => void` ?